src/Pelagos/Bundle/AppBundle/Command/DoiComparisonCommand.php

```diff
@@ -85,8 +85,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     'doi' => $doi['attributes']['doi'],
                     'url' => $doi['attributes']['url'],
                     'udi' => $this->getUdi($doi['attributes']['url']),
-                    'title' => str_replace(',', '', $doi['attributes']['titles'][0]['title']),
-                    'author' => str_replace(',', '', $doi['attributes']['creators'][0]['name']),
+                    'title' => str_replace(
+                        ',',
+                        '',
+                        $this->isArrayEmpty($doi['attributes']['titles'])
```

Comment:
Wouldn't it be better is to use `array_key_exists` that way you can for sure tell if 'name' is there. Not empty doesn't mean the key exists...

Reply:
To check if 'name' exists, I think i need to do a nested array_key_exists